tmt/steps/discover/fmf.py

```diff
@@ -224,15 +226,15 @@ def go(self):
                 "Cannot manipulate with dist-git without "
                 "the `--dist-git-merge` option.")
 
-        def get_git_root(dir):
+        def get_git_root(dir: str) -> Any:
```

Comment:
It has to return Str

Reply:
how do I overcome tmt/steps/discover/fmf.py:230: error: Returning Any from function declared to return "str" ?